test/build-test.js

```diff
@@ -67,11 +68,16 @@ describe('build', function() {
 			var s1 = delay(1, fromArray([1,2]));
 			var s2 = fromArray([3,4]);
 
-			return reduce(function(a, x) {
-				return a.concat(x);
-			}, [], build.concat(s1, s2)).then(function(a) {
-				expect(a).toEqual([1,2,3,4]);
-			});
+			var scheduler = new TestScheduler();
+			scheduler.tick(1);
+
+			return scheduler.collect(build.concat(s1, s2))
```

Comment:
Then, call `collect` to run a stream on the virtual timeline and collect all the events as `{ time, value }` pairs.